action.yml

```diff
@@ -79,6 +82,12 @@ runs:
       with:
         files: "${{ inputs.dockerfile }}"
         allow_failure: true # fails the Action if Dockerfile is missing
+        
+    - name: Install qemu dependency for multi-arch
```

Comment:
Is this step really needed? Does not buildah itseft take care about the needed dependencies for build of multiarch images?

Reply:
That step is required . I will use an action for qemu , but right now i am using it to test it. But step is a must

Reply:
Can you please explain, why it is a must?

Reply:
https://github.com/redhat-actions/buildah-build/tree/main -- This is the github link for the action that is being currently used for both builld & push operation in the workflow. In the github link , it has been specified that qemu-static is a must. You can find it under **Multi arch builds** section

Reply:
Thanks for the explanation. Makes sense. Please provide this information in the final commit message.

Anyway, I think it should be still sufficient to enable this step only when needed - most of the builds will stay at x86_64 only, so the installation will not be needed every time.
The step can be made conditional based on user input similarly to this: https://github.com/sclorg/build-and-push-action/blob/8f488e6fae4bc71bd1131abcf320d551c6ccd687/action.yml#L84